packages/network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+### Changed
+- **BREAKING:** Update type of state object by renaming `properties` property to `networkDetails` ([#1074](https://github.com/MetaMask/core.git/pull/1074))
+  - Consumers are recommended to add a state migration for this change.
+- **BREAKING:** Rename `NetworkProperties` to `NetworkDetails` ([#1074](https://github.com/MetaMask/core.git/pull/1074))
```

Comment:
It looks like this was included twice

```suggestion
```

Reply:
I paired with Nidhi on adding these and suggested that she add this separately from the above entry since that once mentions changing a property within the type that represents the state and this one mentions renaming the type that represents `networkDetails` specifically, but if you think this is implied or unnecessary, no problem.

Reply:
Oh, I thought it was referring to the same state property. Maybe add "type" to the entry somewhere? I didn't realize it was a type.

Reply:
Okay yeah, maybe this?

``` suggestion
- **BREAKING:** Rename `NetworkProperties` type to `NetworkDetails` ([#1074](https://github.com/MetaMask/core.git/pull/1074))
```

Reply:
That works!